daml-script/daml3/BUILD.bazel

```diff
@@ -10,12 +10,24 @@ genrule(
     name = "daml-script",
     srcs = glob(["**/*.daml"]),
     outs = ["daml3-script.dar"],
+    # TODO How on earth do you get the location of the daml directory for a cp -r
```

Comment:
Right I tried this, but getting the paths right is then tricky, since the file structure needs to match from source to dest.
the locations it'll give will be absolute paths, so you'd need to cut off everything up to whatever directory Script.daml is in from every path